### DISCUSSION TITLE
Code not being opened when calling edit. No errors in the log 😱

### DISCUSSION THREAD

**Discussion by kentcdodds** [johnlindquist/kit#206] [Errors] [answered]
Really no idea what's going on, but here's my script:

```js
// Menu: Open Project
// Description: Opens a project in code
// Shortcut: cmd shift .

async function getProjects(parentDir) {
  const codeDir = await ls(parentDir)
  const choices = []
  for (const dir of codeDir) {
    if (dir.includes('node_modules')) continue

    const fullPath = path.join(parentDir, dir)
    if (await isFile(path.join(fullPath, 'package.json'))) {
      choices.push({
        name: dir,
        value: fullPath,
        description: fullPath,
      })
    } else {
      choices.push(...(await getProjects(fullPath)))
    }
  }
  return choices
}

const choice = await arg('Which project?', async () => [
  ...(await getProjects('~/code')),
  ...(await getProjects('~/Desktop')),
])

await edit(choice)
```

This worked up until a day or two ago and now vscode isn't opening that project.

The log has:

```
[2021-04-28 10:11:49.825] [info] > Opening ~/code/remix-kentcdodds with code
```

Interestingly kit just stays open showing this indefinitely:

![image](https://user-images.githubusercontent.com/1500684/116436999-31963900-a80a-11eb-9e7e-21fd28b38859.png)

**Comment by johnlindquist** [comment, marked as answer]
I'll get that sorted this afternoon. I rushed out a build yesterday trying to fix an "Open with Atom" bug which touched on that logic. I'll figure it out

**Reply by kentcdodds** [reply]
Coolio, thanks!

**Reply by johnlindquist** [reply]
Apparently, my dev environment had cached `/usr/local/bin/` into the PATH so it "worked on my machine".
(`code` lives at `/usr/local/bin/code`)

But `edit` couldn't find `code` on _your_ machine since `/usr/local/bin` wasn't on the PATH. I tested on a clean environment (my mac mini) and it looks like it's all sorted. Will do a build in a couple hours. This should fix #196 as well .

**Reply by johnlindquist** [reply]
@kentcdodds plz run a "Help->Check for Update" and let me know if this fixes the issue.

**Reply by kentcdodds** [reply]
Works great. Thanks!
